apps/sms/views/inbox/js/inbox.js

```diff
@@ -646,10 +581,9 @@ var InboxView = {
     }
 
     if (!empty) {
-      TimeHeaders.updateAll('header[data-time-update]');
+      this.list.setModel(this.listData);
```

Comment:
__question:__ wondering if fast list has the ability to set some sort of "empty" template?